src/index.ts

```diff
@@ -106,16 +106,45 @@ export async function run(req: Request, ctx: Helix.UniversalContext): Promise<Re
   let rootPath = devsitePathMatch?.root;
   let path = `${rootPath}${suffixSplitRest.join('/')}`.replaceAll('//', '/');
 
-
-  if(path.endsWith('/')) {
-    // impliclty grab index.md if it's a folder level
-    path += 'index.md';
-    extension = '.md';
+  const getConfig = `https://raw.githubusercontent.com/${ctx.attributes.content.owner}/${ctx.attributes.content.repo}/${ctx.attributes.content.branch}/gatsby-config.js`;
```

Comment:
Hmm we actually want to get this info from the `config.md` because `gatsby-config.js` will not be a thing once the migration is complete. It is easier to extract the path info from the `gatsby-config.js` because it's already in JS rather than parsing it from .md. I do have some logic in the `adp-devsite` repo on how to parse the config.md:
https://github.com/AdobeDocs/adp-devsite/blob/e192d5d8e0bdc74156c2144bd8612db35227ee14/hlx_statics/blocks/side-nav/side-nav.js#L34

https://github.com/AdobeDocs/adp-devsite/blob/e192d5d8e0bdc74156c2144bd8612db35227ee14/hlx_statics/blocks/header/header.js#L375

I'm not sure if there's an easier/better way to do it than the method I'm doing (ie, using `DOMParser` to parse the .md doc and some query selector logic). But if you have an idea, it'd be worth checking out. 

Otherwise, the logic for fixing the paths look great!

Reply:
OKay @timkim , I'll take a look at it